tomviz/ModuleOrthogonalSlice.cxx

```diff
@@ -140,6 +140,9 @@ void ModuleOrthogonalSlice::addToPanel(QWidget* panel)
     << "Slice";
   proxiesWidget->addProxy(this->Representation, "Slice", reprProperties, true);
   proxiesWidget->updateLayout();
+
+  this->connect(proxiesWidget, SIGNAL(changeFinished(vtkSMProxy*)),
+                SLOT(requestRender()));
```

Comment:
I didn't know that.  Neat.  I'm glad they thought of this when designing it since I think I've done very similar things before (maybe not on tomviz, but I don't remember).  I'll redo the branch to do it that way.